apps/web/src/components/pure/loading/page-loading.tsx

```diff
@@ -17,34 +16,9 @@ export const Loading = memo(function Loading() {
   );
 });
 
-// Used for the full page loading
-const StyledLoadingContainer = styled('div')(() => {
-  return {
-    height: '100vh',
-    display: 'flex',
-    justifyContent: 'center',
-    alignItems: 'center',
-    color: '#6880FF',
-    flexDirection: 'column',
-    h1: {
-      fontSize: '2em',
-      marginTop: '15px',
-      fontWeight: '600',
-    },
-  };
-});
-
+/**
+ * @deprecated use skeleton instead
+ */
 export const PageLoading = () => {
-  // We disable the loading on desktop, because want it looks faster.
-  //  This is a design requirement.
-  if (environment.isDesktop) {
-    return null;
-  }
-  return (
-    <StyledLoadingContainer>
-      <Loading />
```

Comment:
We can remove the loading animation since we will all use skeleton in the future.
Just leave it blank